swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.item = item
+        self.inventory.append(item)
+        return self.item
+
+    def remove(self, item):
+        self.item = item
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    def get_by_category(self, category):
+        items_in_inventory = []
+        self.category = category
+        for self.item in self.inventory:
+            if self.item.category == self.category:
+                items_in_inventory.append(self.item)
+        return items_in_inventory
```

Comment:
We can use list comprehension here

```suggestion
    def get_by_category(self, category):
        items_in_inventory = [ item for item in self.inventory if item.category == category]
        return items_in_inventory
```

Reply:
Here's more examples of Python list comprehensions if you want to play around with them 😄 : https://www.w3schools.com/python/python_lists_comprehension.asp